api/EventCounts.json

```diff
@@ -0,0 +1,380 @@
+{
+  "api": {
+    "EventCounts": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": "85"
+          },
+          "chrome_android": {
+            "version_added": "85"
+          },
+          "edge": {
+            "version_added": "85"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "71"
+          },
+          "opera_android": {
+            "version_added": "60"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": "85"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
```

Comment:
For some previous PRs I pushed changes to change this to false for WICG specs. Can you do that for this and other open PRs where the spec URL given starts with https://wicg.github.io/?